### PR TITLE
Configure Matrix configuration task

### PR DESCRIPTION
**Pull request by sherine-k**

### CODE REVIEW

File: manifests/tekton/tasks/base/cosa-send-notification.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: notify-matrix-room
```
Comment (LorbusChris) [inline manifests/tekton/tasks/base/cosa-send-notification.yaml]: could you name this task (and file) `notify-matrix`?

File: manifests/tekton/pipelines/base/okd-coreos-all.yaml [hunk 2]
```diff
@@ -140,6 +146,20 @@ spec:
...
+    - name: notify-matrix
+      params:
+        - name: matrix-access-token
+          value: matrix-access-token
+        - name: endpoint
+          value: $(params.matrix-endpoint)
+        - name: room
+          value: $(params.matrix-room)
+        - name: message
+          value: "$(params.variant) - version $(params.version): Build from branch $(params.branch) $(tasks.status)"
```
Comment (LorbusChris) [inline manifests/tekton/pipelines/base/okd-coreos-all.yaml]: I wonder if we can put a link to the Pipelinerun in here?

File: base/tekton.dev/tasks/notify-matrix.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    image: docker.io/curlimages/curl:7.70.0@sha256:031df77a11e5edded840bc761a845eab6e3c2edee22669fb8ad6d59484b6a1c4 #tag: 7.70.0
+    script: |
+      #!/bin/sh
+      if [[ -z "$(params.room)" || -z "$(params.endpoint)" ]]; then
+        echo "No Matrix parameters found - no notification sent"
+      else
+        /usr/bin/curl -X POST -H 'Content-type: application/json' --data '{"msgtype":"m.text", "body":"$(params.message)"}' https://$(params.endpoint)/_matrix/client/r0/rooms/$(params.room)/send/m.room.message?access_token=$TOKEN
+      fi
+    env:
+    - name: TOKEN
```
Comment (lmzuccarelli) [inline base/tekton.dev/tasks/notify-matrix.yaml:42]: Is this token related to the room id ? - if it is then if we change the room in the params it would fail ?, or does it not apply

**Review by lmzuccarelli — COMMENTED** [review]
Great work @sherine-k

**Review by lmzuccarelli — APPROVED** [review]
